src/FamilyHubs.Referral.Web/Pages/Shared/_TextBoxDetail.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@model FamilyHubs.Referral.Core.Models.PartialTextBoxViewModel
+@if (Model.ValidationValid == false)
+{
+    <div class="govuk-error-summary" data-module="govuk-error-summary">
+        <h2 class="govuk-error-summary__title" id="error-summary-title">
```

Comment:
Could you please check this mark-up against the docs for the [GDS error summary](https://design-system.service.gov.uk/components/error-summary/). E.g. looks like there should be `<div role="alert">` at this level.

Reply:
Added